practice/react-practice-one/src/components/Toasts/Toast.jsx

```diff
@@ -0,0 +1,28 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+
+/* Import CSS */
+import './toast.css';
+
+/* Import constants */
+import { TOAST_TYPES, MESSAGES } from '../../constants';
+
+const Toast = ({ message, type, duration = 3000, onClose }) => {
+  useEffect(() => {
+    const timer = setTimeout(onClose, duration);
+
+    return () => clearTimeout(timer);
+  }, [duration, onClose]);
+
+  return <div className={`toast ${type}`}>{message}</div>;
+};
+
+Toast.propTypes = {
+  message: PropTypes.oneOf(Object.values(MESSAGES)),
```

Comment:
@JennyDoJD I think that making it a required string is enough

Reply:
I fixed it